invoices_app/views.py

```diff
@@ -358,6 +358,12 @@ def change_invoice_status(request, pk):
         )
 
     invoice.save()
+    invoice_statuses_json = {n:m for n, m in INVOICE_STATUS}
```

Comment:
FYI (for your information): This is not a json, it is a python dictionary.

Another comment is that I think that it will be better if you move this dictionary to the constants file and import it from there, also this will help to [this line](https://github.com/evbeda/suppliers_management/pull/331/files#diff-3e9a99f7b67b798d91d67bbc9d222263R340) because you will just put `if status not in invoice_statuses.values()`